lib/Doctrine/Core.php

```diff
@@ -656,7 +656,7 @@ public static function loadModels($directory, $modelLoading = null, $classPrefix
 
                     if (end($e) === 'php' && strpos($file->getFileName(), '.inc') === false) {
                         if ($modelLoading == Doctrine_Core::MODEL_LOADING_PEAR) {
-                            $className = str_replace($dir . DIRECTORY_SEPARATOR, null, $file->getPathName());
+                            $className = str_replace($dir . DIRECTORY_SEPARATOR, '', $file->getPathName());
```

Comment:
`str_replace` technically doesn't accept `null` here (should be an array or string) so just changing it to empty string (which is what null would be coerced to in this case).